CONTRIBUTING.md

```diff
@@ -0,0 +1,224 @@
+    knobs: { // you can run the tests using custom knobs, if the story supports them.
+      large: { Size: 64 }, // "large" is the name of test, it will be used in the name of the snapshot file. "Size" is the label of the knob, e.g. size: { default: number('Size', 48) }
+      'type-error': { Type: 'error' },
+    },
+  },
+}
+```
+
+## Linting
+
+To lint and fix errors in files:
+
+```sh
+npm run lint
+```
+
+## Build
+
+To build your Storybook:
+
+```sh
+npm run build-storybook
+```
+
+## Submitting changes
+
+These are the steps to contribute with a new change to this repository:
+
+### Creating a PR
+
+A **Pull Request** (PR) is the method of submitting contributions to the project. The project uses Github for PR management and can be created from JIRA issues. The PR is a process for peer review by code maintainers and relevant developers involved in the changes. Considerations during the Pull Request creation:
+
+* The **destination branch** of the Pull Request must be `master` branch.
+* The **title** follows the format: [issue-id] type: Title of the issue. (i.e. `[ONL-123] chore: Changing the documentation`) Where the type of the commit follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+* Update the **description** to include a descriptive text for changes with notes for the reviewers and screenshots if necessary.
+* The default **reviewers** must be the code owners defined in the [CODEOWNERS](CODEOWNERS) file and you must include also some code owners related with third party services, when these changes affects them. Share the PR in [#online-reviews](https://app.slack.com/client/TFMC43PH7/C0132DEVAPM) Slack channel to be reviewed by Online team members.
+* Mark the option "close branch after the Pull Request is merged".
+
+### Code Review
+
+**Code Review** is an integral process of software development that helps identify bugs and defects before the testing **phase**. We use the Pull Request (PR) mechanism in Github to do this process. These are the considerations for the code review phase:
+
+* The PR is considered **approved** when **some code owner** have approved the PR.
+* The comments can be done at PR level or at detailed level.
+* The comments can be in different levels of relevance: **mandatory** or **optional**.
+    * The mandatory changes must have a associated **task** in the Pull Request view.
+    * All the mandatory changes must be changed and the associated tasks marked as **done**.
+* The code owners review the manual checks for **Style guide** of the project.
+* If the change is rejected by the code owners and there is not possibility of modifications, the PR will be marked as **declined**, and the process do not continue. If it can be modified, the Pull Request should not be declined, only commented with mandatory changes.
+* The code review phase is considered **finished** when all the mandatory changes are done and it is approved.
+
+### Merge conditions
```

Comment:
@jaimeperezpedra Should we mention if we should follow squash strategy when we are merging?

Reply:
done!